src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,413 @@
+
+#### Background reading
+
+[Nodes and Clients](https://ethereum.org/developers/docs/nodes-and-clients)
+
+#### Client implementations
+
+[Geth client](geth.ethereum.org)
+
+[Nethermind client](nethermind.io)
+
+[Besu client](https://consensys.net/quorum/developers/)
+
+[Erigon client](https://github.com/ledgerwatch/erigon)
+
+#### Recent research
+
+[Database optimizations](https://github.com/ledgerwatch/erigon/blob/devel/docs/programmers_guide/db_faq.md)
+
+[State expiry](https://notes.ethereum.org/@vbuterin/state_expiry_eip)
+
+[History management](https://eips.ethereum.org/EIPS/eip-4444)
+
+[Verkle Trees](https://vitalik.ca/general/2021/06/18/verkle.html)
+
+[Data availability sampling](https://github.com/ethereum/research/wiki/A-note-on-data-availability-and-erasure-coding)
+
+
+### Formal verification
+
+Formal verification is writing code to verify that Ethereum's consensus specifications are correct and bug-free. There is an executable version of the specification written in Python that requires maintenance and development. Further research can help to improve the Python implementation of the specification and add tools that can more robustly verify correctness and identify issues.
+
+#### Background reading
+
+[Introduction to formal verification](https://ptolemy.berkeley.edu/projects/embedded/research/vis/doc/VisUser/vis_user/node4.html)
+
+[Formal Verification (Intel)](https://www.cl.cam.ac.uk/~jrh13/papers/mark10.pdf)
+
+
+#### Recent research
+
+[Formal verification of the deposit contract](https://github.com/runtimeverification/deposit-contract-verification)
+
+[Formal verification of the Beacon Chain specification](https://github.com/runtimeverification/deposit-contract-verification)
+
+
+
+### Data science and analytics
+
+There is a need for more data analysis tools and dashboards that give detailed information about activity on Ethereum and the health of the network.
+
+### Background reading
+
+[Dune Analytics](https://dune.com/browse/dashboards)
+
+[Client diversity dashboard](https://clientdiversity.org/)
+
+#### Recent research
+
+
+
+## Apps and tooling
+
+### DAOs
+
+DAOs are one of the key classes of application being built on top of Ethereum. There is a lot of very active research into how they may develop to replace corporations and other traditional organizations.
+
+#### Background reading
+
+[Dao Collective](https://daocollective.xyz/)
+
+
+#### Recent research
+
+[Mapping the DAO ecosystem](https://www.researchgate.net/publication/358694594_Mapping_out_the_DAO_Ecosystem_and_Assessing_DAO_Autonomy)
+
+### Developer tools
+
+Tools for Ethereum developers are rapidly improving. There is lots of active research and development to do in this general area.
+
+#### Background reading
```

Comment:
Maybe a good start is just listing our dev documentation, like resources by programming language: https://ethereum.org/en/developers/docs/programming-languages/

Reply:
Token standards might be a good resource to list: https://ethereum.org/en/developers/docs/standards/tokens/

Reply:
https://docs.ethhub.io/ethereum-basics/development/tooling/

https://biastek.com/ethereum-tools/

https://cryptodevhub.io/wiki/ethereum-virtual-machine-tools